### DISCUSSION TITLE
Instagram ?!?!

### DISCUSSION THREAD

**Discussion by MarkJaroski** [jakopako/croncert-config#157] [General]
This is extremely annoying: one of the main clubs in Lausanne for concerts by up-and-coming local artists have moved their entire calendar over to Instagram.

https://bleu-lezard.ch/la-cave/

I don't see any work-around for this, at least because of the TOS.

**Comment by jakopako** [comment]
This is indeed annoying. Not only because of the TOS but I also suppose that Instagram is making scraping their website really hard from a technical perspective.

**Reply by MarkJaroski** [reply]
I know there's nothing we can do about it, other than to complain at the venue. So I'm just venting.

I guess I could stand up a scrape-friendly site for them as a gesture of support, and hand enter the data, but that can wait.

**Comment by MarkJaroski** [comment]
This thread: https://github.com/tobilg/facebook-events-by-location-core/issues/29 is worth reading.

It seems that Meta have decided to corner the market on user-contributed events and are not interested in playing nicely.

**Reply by jakopako** [reply]
Interesting read indeed. 

At least regarding concerts it happened very rarely until now that events were only present on FB, luckily. Still, I'm wondering how important FB events still are. I guess a lot of people are using them and I'm just a bit out of touch since I deleted my FB account a couple of years ago.
